### ISSUE TITLE
Word hunt/scramble crash

### ISSUE THREAD

**Issue by pankaka1** [donato-fiore/TrollStoreGameSeagull#3]
Not sure if it’s just me having this issue but word hunt and scramble crashes for me when I press start. It happens only when I have the cheats on for word game but everything else with gameseagull works well.

**Comment by grasscutting** [comment]
Yeah, you need to disable the word game cheats when playing those games unless it will crash :(
